plugins/repository-gcs/build.gradle

```diff
@@ -180,7 +182,7 @@ tasks.named("thirdPartyAudit").configure {
     'org.apache.http.protocol.HttpRequestExecutor',
     // commons-logging provided dependencies
     'javax.servlet.ServletContextEvent',
-    'javax.servlet.ServletContextListener'
+    'javax.servlet.ServletContextListener',
```

Comment:
comma not needed?

Reply:
Got automatically added by IntelliJ, fixing